Modules/ExtendedPlayerControl.cs

```diff
@@ -680,7 +680,7 @@ public static bool HasKillButton(this PlayerControl pc)
             _ => false
         };
     }
-    public static bool CanUseVent(this PlayerControl player) => player.CanUseImpostorVentButton() || player.Data.Role.Role == RoleTypes.Engineer;
+    public static bool CanUseVent(this PlayerControl player) => player.CanUseImpostorVentButton() || player.GetCustomRole().GetVNRole() == CustomRoles.Engineer;
```

Comment:
@Tommy-XL 
Can you check is this change is correct?

Reply:
Yes, but it mostly doesn't make sense since we never use desync engineer

Reply:
Welp this is sussy
The host checks this and it works for vanilla
But im not sure whether previous code is ok for modded clients

So just change it since it doesnt cause bugs